### PR TITLE
[ADD] l10n_kr: Base accounting package

### PR DESCRIPTION
**Pull request by vin-odoo**
Add basic support for South Korean accounting:
- Chart of Account
- Taxes and tax report
- Address format on partner view
- VAT Validation references

Task ID: 4262097

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: odoo/modules/loading.py [hunk 1]
```diff
@@ -97,7 +98,7 @@ def load_demo(env, package, idref, mode):
         Failure = env.get('ir.demo_failure')
         if todo and Failure is not None:
             todo.state = 'open'
-            Failure.create({'module_id': package.id, 'error': str(e)})
```
Comment (kmagusiak) [inline odoo/modules/loading.py:100]: Make it a separate PR?

File: addons/account/models/chart_template.py
Comment (hupo-odoo) [inline addons/account/models/chart_template.py:1]: If I understand correctly, you want to set default accounts for Korean companies like account_production_wip_account_id which are accounts defined in mrp_accountant, but not do a bridge module just to set default accounts, am I right ?

**Review by hupo-odoo — COMMENTED** [review]
Good job! I haven't much to say except a few comments and typos :+1:

File: odoo/modules/loading.py (hunk 1 above)
Comment (vin-odoo) [reply odoo/modules/loading.py:100]: Done here thanks https://github.com/odoo/odoo/pull/189259 in 16 instead
I wasn't sure this change would be acceptable so I was expecting to first check with your team first and I then forgot :see_no_evil:

File: addons/l10n_kr/data/template/account.tax.group-kr.csv
[the remark this answers is not in the document]
Comment (vin-odoo) [reply addons/l10n_kr/data/template/account.tax.group-kr.csv:1]: I quote KIDO

> For the Tax Group, despite there being many different taxes in this PR, they are considered to be one group "VAT" in Korea.
On the invoices and receipt, it is not a common practice also to display how much % is is (They could if they want) but they normally just write VAT.
Lastly, in the end, it is just part of the same group whether it is 10%, 0%, Exempt which is VAT as well and they are managed in one same VAT report. So there is not really a need to create a separate group for the ones we have at the moment. 

If you or Juliette think we should change it, we can of course

**Comment by vin-odoo** [comment]
> I also notice that account groups are present in the xlsx file in the task but missing on the PR, is it on purpose ?

Which group are you talking about? We're not sure about it

File: addons/account/models/chart_template.py
Comment (vin-odoo) [reply addons/account/models/chart_template.py:1]: Yes, I was trying to find a better solution and to bother WAN to check with me but he was swamped :)
Theoretically, the proper way to do this would be to move these accounts to the account module, like account_default_pos_receivable_account_id. 
As I'm targeting 18, it wouldn't be doable.

From my test, defining these accounts already works fine, it's just the test raising an error due to the check on the fields.

I believe we don't want to add bridge modules everywhere to define a few default accounts.

**Comment by hupo-odoo** [comment]
> > I also notice that account groups are present in the xlsx file in the task but missing on the PR, is it on purpose ?
> 
> Which group are you talking about? We're not sure about it

I'm talking about the ones defined in the xlsx file in the first page ("Overview - KIDO") 

For example:
`1 - 자산 - Assets`
`11 - 유동자산 - Current Assets`
...

File: addons/account/models/chart_template.py
Comment (hupo-odoo) [reply addons/account/models/chart_template.py:1]: Thank you very much for the explanation. In this case it looks good to me to do like this. As you say, a cleaner way would be to do it in account but not possible in stable versions.

**Comment by vin-odoo** [comment]
@hupo-odoo I see, I don't think we need these in the COA :)
You can find this structure in the balance sheet though

File: addons/l10n_kr/data/template/account.tax.group-kr.csv
[the remark this answers is not in the document]
Comment (hupo-odoo) [reply addons/l10n_kr/data/template/account.tax.group-kr.csv:1]: I think this explanation is good enough, let's keep it like this if KIDO made the analysis and came to this conclusion :smiley:

**Comment by hupo-odoo** [comment]
> @hupo-odoo I see, I don't think we need these in the COA :) You can find this structure in the balance sheet though

If you think it's not needed I'm fine with it, but account groups are mostly used in reports such as the trial balance or general ledger when activating the hierarchy

**Comment by vin-odoo** [comment]
@hupo-odoo good point, discussed it with kido and we'll be adding the groups a bit later, I'll ping you once it's done :)

**Comment by vin-odoo** [comment]
@hupo-odoo Thank you, I have pushed the addition of the groups :)

**Comment by hupo-odoo** [comment]
@jbw-odoo ready for your review

**Review by jbw-odoo — APPROVED** [review]
robodoo r+
@odoo/rd-runbot possible to clear the ci-template, new module added here and in enterprise https://github.com/odoo/enterprise/pull/74143
